src/FsAutoComplete/LspServers/AdaptiveServerState.fs

```diff
@@ -1437,12 +1452,17 @@ type AdaptiveState
     asyncAVal {
       let wins =
         openFilesToChangesAndProjectOptions
-        |> AMap.map (fun _k v -> v |> AsyncAVal.mapSync (fun (_, projects) _ -> projects))
```

Comment:
The main issue here is the calls to `parseFile` weren't happening after file changes because of how F# Data Adaptive works when caching values. Even if something changes upstream (like file changes), the list of projects was staying the same and causing us to use stale parsechecks.

Reply:
This makes sure to allow the file changes to pass thru so we can use them correctly for updates.